libs/game/game.ts

```diff
@@ -14,11 +71,13 @@ namespace game {
     export let debug = false;
     export let flags: number = 0;
     export let gameOverSound: () => void = undefined;
-    let __eventContext: control.EventContext;
-    let __isOver = false
+
+    export let scene: Scene;
+    let sceneStack: Scene[];
+
+
     let __waitAnyKey: () => void
-    let __background: Background;
-    let __tileMap: tiles.TileMap;
+    let __isOver = false;
```

Comment:
I might have started it, but I'm not sure why we have all these underscores here...